srcs/juloo.keyboard2/Keyboard2.java

```diff
@@ -520,4 +545,66 @@ private View inflate_view(int layout)
   {
     return View.inflate(new ContextThemeWrapper(this, _config.theme), layout, null);
   }
+
+  @RequiresApi(api = Build.VERSION_CODES.R)
+  @Nullable
+  @Override
+  public InlineSuggestionsRequest onCreateInlineSuggestionsRequest(@NonNull Bundle uiExtras)
+  {
+    Size smallestSize = new Size(0, 0);
+    Size biggestSize = new Size(Integer.MAX_VALUE, Integer.MAX_VALUE);
+
+    UiVersions.StylesBuilder stylesBuilder = UiVersions.newStylesBuilder();
+
+    InlineSuggestionUi.Style style = InlineSuggestionUi.newStyleBuilder().build();
+    stylesBuilder.addStyle(style);
+
+    Bundle stylesBundle = stylesBuilder.build();
+    InlinePresentationSpec spec =
+            new InlinePresentationSpec.Builder(smallestSize, biggestSize)
+                    .setStyle(stylesBundle)
+                    .build();
+
+    List<InlinePresentationSpec> specList = new ArrayList<>();
+    specList.add(spec);
+
+    InlineSuggestionsRequest.Builder builder = new InlineSuggestionsRequest.Builder(specList);
+
+    return builder.setMaxSuggestionCount(InlineSuggestionsRequest.SUGGESTION_COUNT_UNLIMITED)
+            .build();
+  }
+
+  @RequiresApi(api = Build.VERSION_CODES.R)
+  @Override
+  public boolean onInlineSuggestionsResponse(@NonNull InlineSuggestionsResponse response)
+  {
+    List<InlineSuggestion> inlineSuggestions = response.getInlineSuggestions();
```

Comment:
The code that manages the autofill container views and manages the reponse should be moved to an other class to avoid making this class bigger.